examples/tests/sdk_tests/function/aws-function.test.w

```diff
@@ -35,4 +36,14 @@ test "validates the AWS Function" {
     // If the test is not on AWS, it should not fail, so I am returning true.
     assert(true);
   }
-}
\ No newline at end of file
+}
+
+if target == "tf-aws" {
+  test "can access lambda context" {
+    let ctx = aws.context();
```

Comment:
```suggestion
    let ctx = aws.Function.context();
```

I think this to be semantically tied to `aws.Function`. 

It should also return an optional and resolve to `nil` on non-AWS functions and in the simulator.

Reply:
Done